FrameworkSystem/Service/SystemAdministratorHandler.py

```diff
@@ -633,8 +632,7 @@ def export_getComponentDocumentation( self, cType, system, module ):
     except Exception as _e:
       return S_ERROR( 'No documentation was found' )
 
-  types_storeHostInfo = []
-  def export_storeHostInfo( self ):
+  def __storeHostInfo( self ):
```

Comment:
result contains the callstack as well. The gLogger.error does not print the callstack. and in the code it is clear where the error comes from